backend-diagram-converter/core/src/main/resources/message-templates.properties

```diff
@@ -43,7 +43,8 @@ connector-id={{ templates.element-transformed-prefix }} Connector id is transfor
 property={{ templates.element-transformed-prefix }} Property '{{ propertyName}}' lives in the Zeebe namespace now.
 execution-listener={{ templates.element-not-transformable-prefix }} Execution Listeners do not exist in Zeebe.
 task-listener={{ templates.element-not-transformable-prefix }} Task Listeners do not exist in Zeebe.
-failed-job-retry-time-cycle={{ templates.element-not-transformable-prefix }} The timecycle '{{ timecycle }}' has to be transformed to Retries.
+failed-job-retry-time-cycle={{ templates.element-transformed-prefix }} The timecycle '{{ timecycle }}' was transformed to '{{ retries }}' retries. Please review. Please keep in mind that the timeout between the retries has to be defined by your worker implementation.
```

Comment:
> Please keep in mind that the timeout between the retries has to be defined by your worker implementation

Why not provide the duration in a header to the worker?

Reply:
This is a good idea.